lib/models/github.dart

```diff
@@ -326,13 +326,47 @@ class GithubUserOrganizationItem {
       _$GithubUserOrganizationItemFromJson(json);
 }
 
+@JsonSerializable(fieldRename: FieldRename.snake)
+class GistFiles {
+  GistFiles({
+    this.filename,
+    this.size,
+    this.rawUrl,
+    this.type,
+    this.language,
+    this.truncated,
+    this.content,
+  });
+  String filename;
+  int size;
+  String rawUrl;
+  String type;
+  String language;
+  bool truncated;
+  String content;
+
+  factory GistFiles.fromJson(Map<String, dynamic> json) =>
+      _$GistFilesFromJson(json);
+}
+
 @JsonSerializable(fieldRename: FieldRename.snake)
 class GithubGistsItem {
-  int id;
-  bool isFork;
-  bool isPublic;
-  String name;
+  String id;
+  String description;
+  bool public;
+  Map<String, GistFiles> files;
+  GithubEventUser owner;
+  List<dynamic> get fileNames {
```

Comment:
```suggestion
  List<GistFiles> get fileNames {
```

Reply:
This was what I started with initially. I got the error `type 'List<dynamic>' is not a subtype of type 'List<GistFiles>'.` That's why I made it dynamic.